lib/checks/keyboard/no-focusable-content-evaluate.js

```diff
@@ -14,7 +15,7 @@ export default function noFocusableContentEvaluate(node, options, virtualNode) {
     }
 
     const notHiddenElements = focusableDescendants.filter(
-      usesUnreliableHidingStrategy
+      vNode => !isInTabOrder(vNode)
```

Comment:
By calling into isFocusable again, this is going to be slower than it needs to be. Best to revert.